### PR TITLE
walletkit+sweeper: add forgotten P2TR address type

### PR DESCRIPTION
**Pull request by guggero**
## Change Description
Fixed three more occurrences of address type enums that didn't yet support P2TR inputs.

NOTE: I didn't add anything to the release notes since this should've been added by any of the previous P2TR pull requests.

### CODE REVIEW

**Review by bhandras — APPROVED** [review]
LGTM :palm_tree:

File: sweep/tx_input_set.go [hunk 1]
```diff
@@ -361,6 +361,8 @@ func createWalletTxInput(utxo *lnwallet.Utxo) (input.Input, error) {
 		witnessType = input.WitnessKeyHash
 	case lnwallet.NestedWitnessPubKey:
 		witnessType = input.NestedWitnessKeyHash
+	case lnwallet.TaprootPubkey:
```
Comment (Roasbeef) [inline sweep/tx_input_set.go:364]: Let's also make the default addresses for the sweeper P2TR as well? 

So we'd change this argument here: https://github.com/lightningnetwork/lnd/blob/master/server.go#L4422-L4424

That'll also indirectly cause all on chain sweeps (force closes, etc) to go into P2TR addresses.

Comment (Roasbeef) [reply sweep/tx_input_set.go:364]: We should also be able to change this as well: https://github.com/lightningnetwork/lnd/blob/de6fcf15b6e090613d62fc859f42b418124730a1/peer/brontide.go#L2162-L2175. 

In practice we _should_ always be using the upfront shutdown addr, but this should be safe.

Comment (Roasbeef) [reply sweep/tx_input_set.go:364]: We'll also need to update this, so older nodes will accept a P2TR addr as an upfront shutdown script: https://github.com/lightningnetwork/lnd/blob/dceb10144f7188a09f5fcdbe04ce4ef9adf6960a/lnwallet/wallet.go#L2247-L2268

I think this means that we shouldn't yet switch over the default co-op close addr due to this stricter validation.

Comment (Roasbeef) [reply sweep/tx_input_set.go:364]: Re the above, here's where we generate our upfront shutdown addr: https://github.com/lightningnetwork/lnd/blob/262591c3331c2170697fe89b8c49ab857db0adf7/funding/manager.go#L1363-L1375

Comment (Roasbeef) [reply sweep/tx_input_set.go:364]: Some other areas that are safe to change:
  * https://github.com/lightningnetwork/lnd/blob/bd5e29ec83186f00e2975af3031b4aedad2bccb4/lnd.go#L460-L461
  * https://github.com/lightningnetwork/lnd/blob/c647d32cf7d3da17e3f94f937e1e5cf6b29e2b2f/rpcserver.go#L1328-L1330
  * https://github.com/lightningnetwork/lnd/blob/dceb10144f7188a09f5fcdbe04ce4ef9adf6960a/lnwallet/wallet.go#L765-L766

**Review by Roasbeef — APPROVED** [review]
LGTM 🐡

**Pull request merged**